lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -335,9 +336,15 @@ int main(int argc, char* argv[]) {
   // every fixed interval of time
   auto periodic_stats_requester = std::make_shared<magma::StatsPoller>();
   std::thread periodic_stats_requester_thread([&]() {
-    //random value assigned for interval period, the value will be loaded
-    //from a config field later
-    periodic_stats_requester->start_loop(local_enforcer, 5);
+    // random value assigned for interval period, the value will be loaded
+    // from a config field later
+    if (config["poll_stats_interval"].IsDefined()) {
+      periodic_stats_requester->start_loop(
+          local_enforcer, config["poll_stats_interval"].as<uint32_t>());
+    } else {
+      periodic_stats_requester->start_loop(
+          local_enforcer, DEFAULT_POLL_INTERVAL_TIME);
+    }
```

Comment:
something like this would be cleaner to read
```suggestion
	uint32_t interval = DEFAULT_POLL_INTERVAL_TIME;
    if (config["poll_stats_interval"].IsDefined()) {
      interval = config["poll_stats_interval"].as<uint32_t>();
    }
    periodic_stats_requester->start_loop(local_enforcer, interval);
```